src/utilities-dom.ts

```diff
@@ -30,11 +30,11 @@ export const getElementDistanceToTopOfDocument = (element: Element): number => {
 };
 
 /**
- * Check if an element has inline style scroll/auto for overflow/overflowY
+ * Check if an element has style scroll/auto for overflow/overflowY
  */
-export const hasOverflowScroll = (element: HTMLElement): boolean => {
-  let style = element.style;
-  return style.overflowY === 'scroll' || style.overflow === 'scroll' || style.overflowY === 'auto' || style.overflow === 'auto';
+export const hasOverflowScroll = (element): boolean => {
+  let style = element !== document ? window.getComputedStyle(element) : undefined;
```

Comment:
- element will never be document. it could be `document.documentElement`.
- because we stop at `document.documentElement` style is never undefined.

Reply:
When searching for the scroller we will react document before null. I did a minor change now, that cleans up ```hasOverflowScroll```